dashboard/assets/packages/ui/src/components/Chart/Chart.hooks.ts

```diff
@@ -41,10 +41,21 @@ export const useOptions = ({ plot, theme, width }: CreateOptionsProps): Options
     }
   }
 
-  const hooks = {
+  const handleRemoveTimeLegend = (uplot: uPlot) => {
+    const legendTable = uplot.root.querySelector('.u-legend')
+    if (legendTable) {
+      const firstRow = legendTable.querySelector('tr')
+      if (firstRow) {
+        firstRow.style.display = 'none'
+      }
+    }
+  }
```

Comment:
By default, the newest version of `uplot` includes a `time` legend. I've decided to remove it via the `ready` hook to keep the UI as close as possible to the previous version.